examples/extending-objects/Abstract Classes/Go/main.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import "fmt"
+
+func main() {
+	marge := AbstractPerson{
```

Comment:
Ensure consistency in documentation across all examples (see what you did here for a comparison: https://github.com/johnfraserss/ICS4U/pull/36/files#diff-ecef7c58901f46a0422a271ea876d9db220e8203902c51fce2ea1226b04624ea )

Reply:
Actually, this is a better example: https://github.com/johnfraserss/ICS4U/pull/36/files#diff-df8fdb95e9a748bd6204cb634d74c8537afba9d91159b413c5c202061c38c07d